operations/commit_queue_test.go

```diff
@@ -142,6 +143,37 @@ func (s *CommitQueueSuite) TestListContentsForCLI() {
 	s.Contains(stringOut, patchURL)
 }
 
+func (s *CommitQueueSuite) TestMergeWithEmptyPatch() {
+	s.Require().NoError(db.ClearCollections(commitqueue.Collection, patch.Collection, model.ProjectRefCollection))
+	now := time.Now()
+	p1 := patch.Patch{
+		Id:          mgobson.ObjectIdHex("aabbccddeeff112233445566"),
+		Project:     "mci",
+		Author:      "annie.black",
+		Activated:   true,
+		Description: "fix things",
+		CreateTime:  now,
+		Status:      evergreen.TaskDispatched,
+	}
+	s.NoError(p1.Insert())
+
+	pRef := &model.ProjectRef{
```

Comment:
do we need a projectRef for this test?

Reply:
I need the mci project in order to create the patch

Reply:
Is there an execution path where we check the project of the patch? Could it be omitted?

Reply:
resolved in person.